conf/VERSION.properties

```diff
@@ -1,4 +1,4 @@
 
 ## Defines the JGroups version
 ## This file should be the *only place* to change when changing the version number
-jgroups.version=3.6.4.Final
\ No newline at end of file
+jgroups.version=4.0.0-SNAPSHOT
```

Comment:
Look at WildFly for instance where conventions tend to be followed first :-)
https://repository.jboss.org/nexus/#nexus-search;gav~org.wildfly~wildfly-parent~~~

Since Bela doesn't often do Alphas/CR for minors/micros, the best name would probably be "3.6.4.Final-SNAPSHOT", for major 4 it could probably start with a alpha so e.g. "4.0.0.Alpha1-SNAPSHOT".